M2/Macaulay2/m2/packages.m2

```diff
@@ -191,7 +191,10 @@ needsPackage String  := opts -> pkgname -> (
     and (opts.FileName === null or
 	realpath opts.FileName == realpath pkg#"source file")
     and pkg.PackageIsLoaded
-    then use value PackageDictionary#pkgname
+    then (
+	if any(packageFiles pkg, file ->fileTime file > filesLoaded#file)
+	then loadPackage(pkgname, opts ++ {Reload => true})
+	else use pkg)
```

Comment:
What's the difference between `use pkg` and `use value PackageDictionary#pkgname`?

Reply:
Nothing -- we'd already defined `pkg` a few lines earlier, so I figured it would be good to use it rather than calling `value` again.